### ISSUE TITLE
Are there any Java examples for reading dhall files?

### ISSUE THREAD

**Issue by parrotcar00** [travisbrown/dhallj#186]
Not familiar with Scala, was wondering if there are any Java examples? I cursorily looked at the repo to find any test files in Java but couldn't find any.

Would be very helpful for intro to this lib.

**Comment by travisbrown** [comment]
Agreed, thanks. I haven't had much time to work on this library lately, but this is something I'll try to prioritize when I do.
